collectd/Dockerfile

```diff
@@ -2,36 +2,30 @@ FROM unit9/base:latest
 
 MAINTAINER Artiom Vasiliev <artiom@unit9.com>
 
-COPY ["configs/sources.list", "/etc/apt/sources.list"]
-COPY ["facette.rc", "/etc/service/facette/run"]
-COPY ["collectd.rc", "/etc/service/collectd/run"]
+COPY "sources.list" "/etc/apt/sources.list"
 
-RUN apt-get update && \
-    apt-get upgrade -y && \
-    apt-get install -y build-essential libgcrypt20-dev librrd-dev snmp \
-        libsnmp-dev snmp-mibs-downloader && \
-    rm -rf /var/cache/apt /var/lib/apt/lists && \
-    cd /tmp && \
+RUN apt-get update \
+    && apt-get install -y \
+        build-essential \
+        libgcrypt20-dev \
+        librrd-dev \
+        snmp \
+        libsnmp-dev \
+        snmp-mibs-downloader \
+    && rm -rf /var/cache/apt /var/lib/apt/lists/* \
+    && cd /tmp \
     # install collectd
-    curl -LO https://storage.googleapis.com/collectd-tarballs/collectd-5.7.1.tar.bz2 && \
-    tar xjf collectd-5.7.1.tar.bz2 && \
-    rm collectd-5.7.1.tar.bz2 && \
-    cd collectd-5.7.1 && \
-    ./configure --with-librrd --enable-librrd --with-snmp --enable-snmp && \
-    make && \
-    make install && \
-    cd /tmp && \
-    rm -r collectd-5.7.1 && \
-    # install facette
-    curl -LO https://github.com/facette/facette/releases/download/0.3.0/facette_0.3.0-1.jessie_amd64.deb && \
-    dpkg -i facette_0.3.0-1.jessie_amd64.deb && \
-    rm facette_0.3.0-1.jessie_amd64.deb && \
-    mkdir /var/run/facette/ && \
+    && curl -LO https://storage.googleapis.com/collectd-tarballs/collectd-5.7.1.tar.bz2 \
+    && tar xjf collectd-5.7.1.tar.bz2 \
+    && rm collectd-5.7.1.tar.bz2 \
+    && cd collectd-5.7.1 \
+    && ./configure --with-librrd --enable-librrd --with-snmp --enable-snmp \
+    && make \
+    && make install \
+    && cd /tmp \
+    && rm -r collectd-5.7.1 \
     # cleanup
-    apt-get remove --purge -y build-essential && \
-    apt-get autoremove -y
+    && apt-get remove --purge -y build-essential \
+    && apt-get autoremove -y
 
-COPY ["configs/facette.json", "/etc/facette/facette.json"]
-COPY ["configs/providers_collectd.json", "/etc/facette/providers/collectd.json"]
-
-EXPOSE 80
+CMD /opt/collectd/sbin/collectd -C /opt/collectd/etc/collectd.conf -f
```

Comment:
1. we don't know what any of the (possibly buggy) plugins may want to do
2. `make run` on the built image; getting: `collectd[7]: plugin_load: plugin "syslog" successfully loaded.`. After `docker exec`ing a `socklog` in the container and sending a SIGINT to collectd, it logged diagnostic messages to syslog before exiting. It will almost certainly log other stuff there as well.

My point is, this is all classical unixy software we're putting in these containers. It has some expectations about its runtime environment. Docker is too cool for that, so we had to build a base image. Runit solves all of these problems, right there in `unit9/base`. And you'd rather ignore it and solve these same problems from scratch?